resource/storage.go

```diff
@@ -155,7 +155,7 @@ func (s storageWrapper) MultiGet(ctx context.Context, ids []interface{}) (items
 }
 
 // Find tries to use storer MultiGet with some pattern or Find otherwise
-func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage int) (list *ItemList, err error) {
+func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage, offset int) (list *ItemList, err error) {
```

Comment:
If adding `offset`, can `page` actually be removed, and then let the REST API code supply offset = page*perPage when page is specified in the request? Or is there arreason that won't work?

It's not very clean code to supply more parameters than needed...

Also, if replacing page with offset, is perPage better renamed to limit to avoid confusion?

The comment applies to all related changes.